trafficFlowTests.py

```diff
@@ -166,6 +173,10 @@ def _run(self, connections: dict, test_type: TestType, test_id: int, index: int,
                     self._enable_measure_cpu_plugin(monitors, node_server_name, node_client_name, True)
                 if plugins['name'] == "measure_power":
                     self._enable_measure_power_plugin(monitors, node_server_name, node_client_name, True)
+                if plugins['name'] == "validate_offload":
+                    iperf_server = servers[-1]
```

Comment:
I think we need to put a TODO here. You are taking the last server at the end of the queue. But we should be running this on each individual server + client pairs.